src/Runner.Worker/Handlers/Handler.cs

```diff
@@ -150,6 +118,40 @@ public override void Initialize(IHostContext hostContext)
             ActionCommandManager = hostContext.CreateService<IActionCommandManager>();
         }
 
+        protected string GetActionRef()
```

Comment:
this method is only get moved down and changed from `public virtual` to `protected`